src/science.md

```diff
@@ -5,9 +5,18 @@
 | [Creating complex numbers][ex-creating-complex-numbers] | [![num-badge]][num] | [![cat-science-badge]][cat-science] |
 | [Adding complex numbers][ex-adding-complex-numbers] | [![num-badge]][num] | [![cat-science-badge]][cat-science] |
 | [Mathematical functions on complex numbers][ex-mathematical-functions] | [![num-badge]][num] | [![cat-science-badge]][cat-science] |
+| [Calculating the side length of a triangle][ex-calculating-side-length-of-triangle] | [![std-badge]][std] | [![cat-science-badge]][cat-science] |
```

Comment:
can you git checkout origin/master -- src/science.md and the other science files and push this?  I need clean commit history